### ISSUE TITLE
Assembler support announcement for x86

### ISSUE THREAD

**Issue by pablodelara** [intel/isa-l#297]
The library currently supports NASM and YASM assemblers for x86. Given that YASM has not been updated in 5 years, we have decided to drop the support for it.
Also, the minimum NASM version supported will be 2.14.01, which has all the instruction set needed in the library. This version was released at the end of 2018, and versions equal or higher than 2.14.01 are available in most Linux distributions.

So in a nutshell, only NASM >= 2.14.01 assembler will be supported from the next release.

**Comment by pablodelara** [comment]
This is already merged.
